collectors/0/iostat.py

```diff
@@ -68,10 +68,14 @@
 import sys
 import time
 import os
+from os.path import basename
 import re
 import copy
+import subprocess
+import errno
+import glob
 
-from collectors.lib import utils
+import utils
```

Comment:
This is related also to the environment where i run an internal fork of tcollector... (don't ask me why...)
So indeed it is wrong here... my bad...